src/Player/Player.cpp

```diff
@@ -34,6 +35,111 @@ namespace Player {
 
     void Player::fortify() {
         Terminal::debug("Player fortify");
+        Terminal::print("Fortification phase");
+        vector<string> selections;
+        bool validChoice = false;
+        int source_country_index;
+        Country * source_country;
+
+        bool can_fortify = false;
+        // confirm fortification is possible
+        for (auto & country : *countries) {
+            for (auto & neighbor : *country->get_neighbors()) {
+                if (this == neighbor->get_owner() && country->get_armies() > 1) {
+                    can_fortify = true;
+                    break;
+                }
+            }
+            if (can_fortify) {
+                break;
+            }
+        }
+
+        if (!can_fortify) {
+            Terminal::print("Fortification is not possible, you do not own two valid neighboring countries");
+            return;
+        } else {
+            Terminal::print("Fortification is possible, would you like to fortify?");
+            vector<string> fortify;
+            fortify.emplace_back("Yes");
+            fortify.emplace_back("No");
+            int response = Terminal::print_select(fortify);
```

Comment:
If you don't send an vector(string), and just send your question (string) to Terminal::print_select, the terminal will actually ask a Y/N question and return a boolean.

Reply:
Yes but for processing a list of responses, I need to send a vector of integers, so I thought it would be easier to manage it this way for tests, we can also change the Y/N method to accept a 1 as a response as well

Reply:
Not sure if I understand the issue, but yeah it could easily be changed to accept 1 as a yes too!